packages/expo-apple-authentication/src/AppleAuthentication.types.ts

```diff
@@ -213,6 +213,7 @@ export enum AppleAuthenticationUserDetectionStatus {
 export enum AppleAuthenticationButtonType {
   SIGN_IN = 0,
   CONTINUE = 1,
+  SIGN_UP = 2, // requires iOS 13.2 or later
```

Comment:
Might be better to put this information as tsdoc comment, so people are more likely to see this when they use it

Reply:
Good suggestion, thanks!